eZ/Bundle/PlatformInstallerBundle/src/Installer/DbBasedInstaller.php

```diff
@@ -35,7 +35,7 @@ public function setOutput($output)
     protected function copyConfigurationFile($source, $target)
     {
         $fs = new Filesystem();
-        $fs->copy($source, $target);
+        $fs->copy($source, $target, true);
```

Comment:
See https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Filesystem/Filesystem.php#L38

Imho If we add the option we should go for true by default.